frontend/src/screens/settings/DebugTools.tsx

```diff
@@ -304,7 +304,7 @@ export default function DebugTools() {
               Get Network Graph
             </Button>
           </AlertDialogTrigger>
-          {info?.backendType === "LDK" && (
+          {(info?.backendType === "LDK" || info?.backendType === "CASHU") && (
```

Comment:
```suggestion
          {(info?.backendType === "LDK" || info?.backendType === "CASHU" /* TODO: move to generic node command */) && (
```

Reply:
We probably can wait for the generic node command feature to be merged and then undo this actually, then merge this PR